app/lib/dartdoc/backend.dart

```diff
@@ -265,8 +266,35 @@ class DartdocBackend {
     await _deleteAllWithPrefix(prefix, concurrency: concurrency);
   }
 
+  /// Schedules the garbage collection of the [package] and [version].
+  /// The wait queue is in-memory, it is not persisted.
+  void scheduleGC(String package, String version) {
```

Comment:
Let's clearly document that this only does a `best-effort` attempt at cleanup. Since `_gcTasks` is not persistent.